src/libraries/System.Diagnostics.Process/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: ActiveIssue("Disabled due to a runtime crash.", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
This is serious and needs tracking issue

Reply:
Yeah, I didn't filed an issue in advanced in order to check that this disabling helps here.